### DISCUSSION TITLE
mergeAll - doc question

### DISCUSSION THREAD

**Discussion by avrim-tr** [ReactiveX/rxjs#6908] [Q&A]
Hey,

While I was reading the doc about "mergeAll" operator, it says like this:
.....The output Observable only completes once all inner Observables have completed....

Now, I am sure it is not enough to get a complete event once all inner observable "completes", but also the source (high level observable), should complete too.

A [simple example](https://stackblitz.com/run?devtoolsheight=50) which is attached in the site shows it

BTW, for switchAll, it is said:

.....
The resulting observable returned by switchAll will only complete if the source observable completes, and any currently subscribed to inner observable also has completed,
.....

**Comment by jakovljevic-mladen** [comment]
Yes, it looks like this is an issue with the docs. I have created a PR that addresses this issue (not only to `mergeAll`, but `concatAll`, `exhaustAll` and `switchAll` operators as well). Please take a look: #7106

Thanks for reporting.
